backends/advanced/src/advanced_omi_backend/services/mcp_server.py

```
@@ -0,0 +1,532 @@
+            logger.info(f"Resolved email '{identifier}' to user_id: {user.id}")
+            return str(user.id)
+
+        # If not found by email, assume it's already a user_id
+        # Verify it exists
+        from bson import ObjectId
+        try:
+            user = await User.find_one(User.id == ObjectId(identifier))
+            if user:
+                logger.info(f"Verified user_id: {identifier}")
+                return str(user.id)
+        except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)